codec/src/test/java/io/netty/handler/codec/DefaultHeadersTest.java

```diff
@@ -440,6 +440,21 @@ public void iteratorSetValueShouldChangeHeaderValue() {
         assertFalse(headers.contains(of("name1"), of("value3")));
     }
 
+    @Test
+    public void testEntryEquals() {
+        Map.Entry<CharSequence, CharSequence> same1 = newInstance().add("name", "value").iterator().next();
+        Map.Entry<CharSequence, CharSequence> same2 = newInstance().add("name", "value").iterator().next();
+        assertEquals(same1, same2);
```

Comment:
@ryu1-sakai can you also add an assert for the hashcode ?

Reply:
Simply followed the `Map.Entry.hashCode()` specification
https://docs.oracle.com/javase/9/docs/api/java/util/Map.Entry.html#hashCode--